srcjs/output_binding_image.js

```diff
@@ -225,6 +225,18 @@ outputBindings.register(imageOutputBinding, 'shiny.imageOutput');
 
 var imageutils = {};
 
+imageutils.disableDrag = function($img) {
+  // Make image non-draggable (Chrome, Safari)
+  $img.css('-webkit-user-drag', 'none');
+
+  // Firefox, IE<=10
+  // First remove existing handler so we don't keep adding handlers.
+  $img.off('dragstart.image_output');
+  $img.on('dragstart.image_output', function() { return false; });
+
+  // Disable selection of image and text when dragging in IE<=10
+  $el.on('selectstart.image_output', function() { return false; });
+};
```

Comment:
Is this `$el` defined? `yarn build` gives me 

```
../srcjs/output_binding_image.js
  241:3  warning  '$el' is not defined  no-undef

✖ 1 problem (0 errors, 1 warning)
```

Reply:
Shoot, I copied and pasted that part and forgot to change it. I'll fix it.